src/models/user.js

```diff
@@ -54,7 +54,7 @@ export default (sequelize, DataTypes) => {
   }
 
   // hooks
-  User.beforeCreate(User.hashPasswordHook.bind(User))
+  User.beforeValidate(User.hashPasswordHook.bind(User))
```

Comment:
Are you sure that `beforeValidate` is the right hook to hash a password? I think you should be using `beforeCreate`.

cc @abelosorio